src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs

```diff
@@ -0,0 +1,161 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using SFA.DAS.Api.Common.AppStart;
+using SFA.DAS.Api.Common.Configuration;
+using SFA.DAS.ProviderRequestApprenticeTraining.Api.AppStart;
+using SFA.DAS.ProviderRequestApprenticeTraining.Application.Queries.GetEmployerRequest;
+using SFA.DAS.SharedOuterApi.AppStart;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Infrastructure.HealthCheck;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Provider.DfeSignIn.Auth.Application.Queries.ProviderAccounts;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Api
+{
+    [ExcludeFromCodeCoverage]
+    public class Startup
+    {
+        private readonly IWebHostEnvironment _env;
+        private readonly IConfiguration _configuration;
+
+        public Startup(IConfiguration configuration, IWebHostEnvironment env)
+        {
+            _env = env;
+            _configuration = configuration.BuildSharedConfiguration();
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSingleton(_env);
+            services.AddConfigurationOptions(_configuration);
+
+            if (!_configuration.IsLocalOrDev())
```

Comment:
@ChrisJWoodcock do you need 2 if statements for the `if (!_configuration.IsLocalOrDev()`) checks can you not do one if statement?